### PR TITLE
Small text improvement for the support page.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

**Comment by Mrt134** [comment]
rebuild

**Comment by a-stjohn** [comment]
rebuild

File: src/app/pages/system/support/support.component.ts [hunk 1]
```diff
@@ -47,7 +47,20 @@ export class SupportComponent  {
       type: 'paragraph',
       name: 'support_text',
       paraText: this.sanitizer.bypassSecurityTrustHtml(
-        "Search https://redmine.ixsystems.com/projects/freenas to ensure the issue has not already been reported before filing a bug report or feature request. If an issue has already been created, add a comment to the existing issue. Please visit http://www.ixsystems.com/storage/ for enterprise-grade storage solutions and support. <br><br> A Redmine account is required to file an issue. Make an account at https://redmine.ixsystems.com/account/register. Use a valid email address when registering to receive issue status updates.")
+        'Search the <a \
+         href="https://redmine.ixsystems.com/projects/freenas" \
+         target="_blank" style="text-decoration:underline;"> \
```
Comment (wblock) [inline src/app/pages/system/support/support.component.ts]: Don't break lines around an HTTP target.  That leaves whitespace in the link.
So:
style="text-decoration:underline;">FreeNAS issue tracker</a>\

File: src/app/pages/system/support/support.component.ts [hunk 2]
```diff
@@ -47,7 +47,20 @@ export class SupportComponent  {
...
+         FreeNAS issue tracker</a> to ensure the issue has not already \
+         been reported before filing a bug report or feature request. If \
+         an issue has already been created, add a comment to the \
+         existing issue. Please visit the <a \
+         href="http://www.ixsystems.com/storage/" target="_blank" \
+         style="text-decoration:underline;">iXsystems storage page</a> \
+         for enterprise-grade storage solutions and support. <br><br> \
+         <a href="https://redmine.ixsystems.com/account/register" \
+         target="_blank" style="text-decoration:underline;"> \
```
Comment (wblock) [inline src/app/pages/system/support/support.component.ts]: As above, this type of line break adds whitespace in the link, which is bad.

**Comment by erinix** [comment]
rebuild

**Comment by erinix** [comment]
rebuild

**Comment by erinix** [comment]
rebuild

**Comment by erinix** [comment]
rebuild

**Comment by erinix** [comment]
rebuild

**Comment by erinix** [comment]
rebuild

File: src/app/pages/system/support/support.component.ts [hunk 3]
```diff
@@ -47,7 +47,18 @@ export class SupportComponent  {
...
+         style="text-decoration:underline;">FreeNAS issue tracker</a>
+         to ensure the issue has not already been reported before \
+         filing a bug report or feature request. If an issue has \
+         already been created, add a comment to the existing issue. \
+         Please visit the <a href="http://www.ixsystems.com/storage/" target="_blank" \
+         style="text-decoration:underline;">iXsystems storage page</a> \
+         for enterprise-grade storage solutions and support.<br><br> \
+         <a href="https://redmine.ixsystems.com/account/register" target="_blank" \
+         style="text-decoration:underline;">Create a Redmine account</a> to file an issue. Use a valid \
+         email address when registering to receive issue status updates.')
```
Comment (erinix) [inline src/app/pages/system/support/support.component.ts]: There appears to be a problem with this, seems your ' quote ends improperly causing this PR to not build

Comment (wblock) [reply src/app/pages/system/support/support.component.ts]: rebuild

**Pull request merged**